RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -650,3 +754,40 @@ void auto_handleJillOfAllTrades()
 
 	return;
 }
+
+boolean auto_haveBurningLeaves()
+{
+	return auto_is_valid($item[A Guide to Burning Leaves]) && get_campground() contains $item[A Guide to Burning Leaves];
+}
+
+boolean auto_burnLeaves()
+{
+	if (!auto_haveBurningLeaves())
+	{
+		return false;
+	}
+	if (available_amount($item[rake]) < 1)
+	{
+		// visit the pile of burning leaves to grab the rakes
+		visit_url("campground.php?preaction=leaves");
+	}
+	if (item_amount($item[inflammable leaf]) > 73 && !(get_campground() contains $item[forest canopy bed]) && get_dwelling() != $item[big rock] && auto_haveCincho())
```

Comment:
Why not get canopy bed even if we don't have cincho? More free rests and stats are nice for everyone

Reply:
I want to add more use of burning leaves like the free fights & resistance potions etc.
74 leaves is a lot if you don't have any "good" use for them other than restoring HP & MP & the stats are pretty meh since we're spoiled by trainset.